src/wallet/coinselection.h

```diff
@@ -200,21 +200,32 @@ struct OutputGroup
  */
 [[nodiscard]] CAmount GetSelectionWaste(const std::set<CInputCoin>& inputs, CAmount change_cost, CAmount target, bool use_effective_value = true);
 
+enum SelectionAlgorithm : uint8_t
+{
+    BNB = 0,
+    KNAPSACK = 1,
+    SRD = 2,
+    MANUAL = 3,
+};
+
```

Comment:
There is any motivation to use the plain `enum` here and not the `enum class`?

Maybe I'm missing some things but usually, the enum class is preferred in C++.

Reply:
Done